### PR TITLE
Bump versions for v0.28.0

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Detailed description**:
- Bump versions to v0.28.0
- Enable persisting of scheduled transactions

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
CircleCI has a bug which fails to checkout the `application*.yml` files from the branch under test and continues to use the version from master.

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

**Review by Nana-EC — DISMISSED** [review]
LGTM

**Review by ijungmann — DISMISSED** [review]

**Review by xin-hedera — DISMISSED** [review]

**Review by Nana-EC — DISMISSED** [review]
LGTM

**Review by xin-hedera — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -232,6 +232,7 @@ jobs:
       - *docker_redis
     steps:
       - checkout
+      - run: apt-get update && apt-get install -y git && git checkout -- .
```
Comment (Nana-EC) [inline .circleci/config.yml:235]: nit: could have declared this once in `commands`
No biggie since we since it might be short lived anyway

**Review by Nana-EC — APPROVED** [review]
LGTM

**Pull request merged**